src/NServiceBus.Core/Pipeline/PipelineComponent.cs

```diff
@@ -13,11 +13,9 @@ public PipelineComponent(SettingsHolder settings)
             PipelineSettings = new PipelineSettings(modifications, settings);
         }
 
-        public void Initialize(IBuilder builder, IConfigureComponents container)
+        public void Initialize(IBuilder builder)
```

Comment:
Should this also be changed to take `ContainerComponent`?

Reply:
👍

Reply:
Looks like this one might be a bit tricky to do because of `CreateRootContext`. Seems like something to change in a separate PR instead.